examples/hosted-app-action-templates/typescript/contentful-app-manifest.json

```diff
@@ -1,6 +1,7 @@
 {
   "actions": [
     {
+      "id": "example",
       "name": "Example App Action",
       "category": "Custom",
       "description": "This is an example to help you learn how hosted app actions work.",
```

Comment:
small thing; can you add
```allowNetworks: [] ``` to the manifest at the example-action (no validation needed here)?

Reply:
we are already doing validation for allowedNetworks in CCA and attaching the `[]` so I think no need here.

Reply:
No, I meant it to add it to this as example.. We have all other props here but not allowNetworks, so why not add it here

Reply:
for improved DX :)

Reply:
Yeah, I was also thinking about DX and you are right, I will raise another PR for that